src/Structs/Track.js

```diff
@@ -0,0 +1,13 @@
+export default class Track {
+  constructor(index, trackArr) {
```

Comment:
`trackArr` is a strange input value. This should be a constructor which only takes input values and sets them as attributes. Anything to cast a track array to a track object should be outside of the constructor (e.g. class method)